packages/core/upload/admin/src/hooks/useUpload.js

```diff
@@ -24,16 +24,17 @@ const uploadAsset = (asset, folderId, cancelToken, onProgress) => {
     })
   );
 
-  return axiosInstance({
-    method: 'post',
-    url: endpoint,
-    headers: {},
-    data: formData,
-    cancelToken: cancelToken.token,
-    onUploadProgress({ total, loaded }) {
-      onProgress((loaded / total) * 100);
-    },
-  }).then((res) => res.data);
+  return axiosInstance
```

Comment:
if you want you can use the getFetchClient or the useFetchClient otherwise I will replace it with my PR

Reply:
Don't worry I can do it because I have to modify the `fetchClient` function